tests/cli/test_vehicle.py

```diff
@@ -129,6 +129,13 @@
         "HVAC status        on\n"
         "-----------------  -------------------------\n"
     ),
+    "zoe_40.1_json.json": (
+        "{\"battery-status\": {\"timestamp\": \"2020-11-17T09:06:48+01:00\", \"batteryLevel\": 50, \"batteryAutonomy\": 128, \"batteryCapacity\": 0, \"batteryAvailableEnergy\": 0, \"plugStatus\": 0, \"chargingStatus\": -1.0}, "
+        "\"charge-mode\": {\"chargeMode\": \"always\"}, "
+        "\"cockpit\": {\"totalMileage\": 49114.27 }, "
+        "\"res-state\": {\"details\": \"Stopped, ready for RES\", \"code\": \"10\"}, "
+        "\"hvac-status\": {\"externalTemperature\": 8.0, \"hvacStatus\": \"off\"}}"
+    ),
```

Comment:
Very close... one extra space and one carriage return missing at the end.
I've also split it to keep under about 88 chars long.
```suggestion
    "zoe_40.1_json.json": (
        '{"battery-status": {"timestamp": "2020-11-17T09:06:48+01:00", '
        '"batteryLevel": 50, "batteryAutonomy": 128, "batteryCapacity": 0, '
        '"batteryAvailableEnergy": 0, "plugStatus": 0, "chargingStatus": -1.0}, '
        '"charge-mode": {"chargeMode": "always"}, '
        '"cockpit": {"totalMileage": 49114.27}, '
        '"res-state": {"details": "Stopped, ready for RES", "code": "10"}, '
        '"hvac-status": {"externalTemperature": 8.0, "hvacStatus": "off"}}\n'
    ),
```